source/user-manual/api/rbac/configuration.rst

```diff
@@ -92,13 +93,17 @@ The response after adding the new policy should be similar to the one below. The
       "message": "Policy created correctly"
     }
 
-We can query the policy ID at any time, along with the other information, using the endpoint ``GET /security/policies``. For a complete list of resources and actions, please visit :ref:`RBAC reference <api_rbac_reference>`.
+This will create a policy with permission to read information related to agents ``001``, ``002``, ``003`` and ``004``. Additional policies can be created as long as they are not duplicated. Any policy could be modified at any given time if needed, so for example new agents could be added to an existing policy.
```

Comment:
```suggestion
This will create a policy with permission to read information related to agents with ids ``001``, ``002``, ``003`` and ``004``. Additional policies can be created as long as they are not duplicated. Any policy could be modified at any given time if needed, so for example new agents could be added to an existing policy.
```